scripts/compile_cost_assumptions.py

```diff
@@ -1641,7 +1868,7 @@ def add_mean_solar_rooftop(data):
 
     data = add_manual_input(data)
     # add costs for home batteries
-    data = add_home_battery_costs(data)
+    # data = add_home_battery_costs(data)
```

Comment:
Why did you comment this part out? Do you include costs for home batteries? If so, how do they compare to the current ones? Then there should be a switch in case your storage data is not used that the home batteries are still added.

Reply:
Interesting. The store prices almost match, while the power connection prices are much lower (~factor 3) when we consider **non-home** batteries.

### Li-Ion battery from PNNL
![image](https://github.com/PyPSA/technology-data/assets/61968949/946a5394-6584-46f6-b488-63fd9d532d8f)
### Home battery
![image](https://github.com/PyPSA/technology-data/assets/61968949/39318edf-9777-4bc5-b6ce-810c2b37f233)

Regarding implementation, I suggest to keep the home batteries in by default. So we can avoid a switch and people can filter out the relevant data afterwards

Reply:
Here are the switch options: 
![image](https://github.com/PyPSA/technology-data/assets/61968949/7ae23e03-c618-4cb5-a0a7-e995f4298046)
I tested turning off all & turning on all storage. Works.